njit_fastmath.py

```diff
@@ -0,0 +1,72 @@
+import pathlib
+
+from utils import check_callees, check_functions
+
+stumpy_path = pathlib.Path(__file__).parent / "stumpy"
+filepaths = sorted(f for f in pathlib.Path(stumpy_path).iterdir() if f.is_file())
+
+all_functions = {}
+
+ignore = ["__init__.py", "__pycache__"]
+for filepath in filepaths:
+    file_name = filepath.name
+    if file_name not in ignore and str(filepath).endswith(".py"):
+        prefix = file_name.replace(".py", "")
```

Comment:
Similarly, assuming that `file_name` is already a `pathlib` object, you can access the "prefix" via:

```
prefix = file_name.stem
```